tests/installation/welcome.pm

```diff
@@ -34,6 +34,7 @@ sub run() {
             send_key_until_needlematch 'scc-invalid-url-deleted', 'backspace';
             type_string get_var('SCC_URL_VALID');
             wait_still_screen 2;
+            save_screenshot;     # create a needle from this, then replace to an assert_screen
```

Comment:
I guess you can just make it an assert_screen. Then make sure the PR is merged, trigger a test, go into interactive mode, create the needle, reload, continue, done